python/ray/dag/compiled_dag_node.py

```diff
@@ -2235,7 +2238,11 @@ async def execute_async(
         return fut
 
     def visualize(
-        self, filename="compiled_graph", format="png", view=False, return_dot=False
+        self,
+        filename="compiled_graph",
+        format="png",
+        view=False,
+        return_dot=False,
```

Comment:
Can we introduce a `verbose` arg to control the edge details display behavior?

Reply:
added verbose_edges flag, should both type hint and output channel type be behind that, or just output channel type?

Reply:
Let's call it `channel_details` and only use it control the channel information but not type hints

Reply:
updated, also I see that there's these inner channels inside CompositeChannel like IntraProcessChannel and BufferedSharedMemoryChannel, should we show that as well?

Reply:
Yes, those should be shown. We also would like to show the channel IDs.
If the information is too long, we should support multiple levels of verbosity.